tests/bgp/test_bgp_policy.py

```diff
@@ -0,0 +1,554 @@
+          .format(gather_info['cli_options'], gather_info['dut_asn'])
+    logger.debug(gather_info['duthost'].shell(cmd, module_ignore_errors=True))
+    cmd = 'vtysh{} -c "config" -c "router bgp {}" -c "address-family ipv4 unicast" \
+           -c "neighbor {} route-map MATCH_COMMUNITY_RM out" \
+           -c "address-family ipv6 unicast" -c "neighbor {} route-map MATCH_COMMUNITY_RM out"' \
+          .format(gather_info['cli_options'], gather_info['dut_asn'], gather_info['neigh_ip_v4'],
+                  gather_info['neigh_ip_v6'])
+    logger.debug(gather_info['duthost'].shell(cmd, module_ignore_errors=True))
+    logger.debug(gather_info['duthost'].shell('vtysh{} -c "clear bgp * soft"'.format(gather_info['cli_options']),
+                                              module_ignore_errors=True)['stdout'])
+    time.sleep(10)
+
+    verify_rm(gather_info['duthost'], gather_info['neigh_ip_v4'], gather_info['cli_options'], "ipv4",
+              "TWO_PREFIX_RM_V4", "TWO_PREFIX_V4", "3")
+    verify_rm(gather_info['duthost'], gather_info['neigh_ip_v6'], gather_info['cli_options'], "ipv6",
+              "TWO_PREFIX_RM_V6", "TWO_PREFIX_V6", "3")
+
+    # remove match community route map
+    cmd = 'vtysh{} -c "config" -c "router bgp {}" -c "address-family ipv4 unicast" \
+           -c "no neighbor {} route-map MATCH_COMMUNITY_RM out" \
+           -c "address-family ipv6 unicast" -c "no neighbor {} route-map MATCH_COMMUNITY_RM out" -c \
+           "no route-map MATCH_COMMUNITY_RM permit 10" -c "exit" -c "no route-map MATCH_COMMUNITY_RM permit 10"' \
+          .format(gather_info['cli_options'], gather_info['dut_asn'], gather_info['neigh_ip_v4'],
+                  gather_info['neigh_ip_v6'])
+    logger.debug(gather_info['duthost'].shell(cmd, module_ignore_errors=True))
+    logger.debug(gather_info['duthost'].shell('vtysh{} -c "clear bgp * soft"'.format(gather_info['cli_options']),
+                                              module_ignore_errors=True)['stdout'])
+    time.sleep(10)
+    check_baseline(gather_info['duthost'], gather_info['neigh_ip_v4'], gather_info['neigh_ip_v6'], num_v4_routes,
+                   num_v6_routes)
+
+    # Configure and verify Community-list policy to permit only first two routes using regexp
+    cmd = 'vtysh{} -c "show bgp ipv4 unicast community 65001:1"'.format(gather_info['cli_options'])
+    output = gather_info['duthost'].shell(cmd, module_ignore_errors=True)['stdout']
+    assert "Displayed  2 routes and " in output
+    cmd = 'vtysh{} -c "show bgp ipv6 unicast community 65001:1"'.format(gather_info['cli_options'])
+    output = gather_info['duthost'].shell(cmd, module_ignore_errors=True)['stdout']
+    assert "Displayed  2 routes and " in output
+
+    cmd = 'vtysh{} -c "config" -c "router bgp {}" \
+           -c "bgp community-list expanded MATCH_COMMUNITY_RGX_1 permit .*.*.*.*1:1" -c \
+            "bgp community-list expanded MATCH_COMMUNITY_RGX_2 permit .*.*.*.*2:2"' \
```

Comment:
why do we need ".*.*.*.*" here? Shouldn't it be "...." for the regular expression?

Reply:
I updated the regex.